src/nupic/algorithms/SDRClassifier.hpp

```diff
@@ -33,20 +34,30 @@
 #include <string>
 #include <vector>
 
-#include <nupic/types/ClassifierResult.hpp>
 #include <nupic/types/Types.hpp>
 #include <nupic/types/Serializable.hpp>
 
 namespace nupic {
 namespace algorithms {
 namespace sdr_classifier {
-using namespace std;
-using nupic::types::ClassifierResult;
+
 
 const UInt sdrClassifierVersion = 2;
 
 /**
- * 2d map used to store the data. 
+ * PDF - Probability Density Function, distribution of likelihood of values
+ */
+using PDF = std::vector<Real64>;
+
+/**
+ * The key is the step, for predicting multiple time steps into the future.
+ * The key -1 contains the estimate of the actual values.
+ * The value is a PDF of the result being in each bucket.
+ */
+using ClassifierResult = std::map<Int, PDF>;
```

Comment:
nice use of using :+1: 
consider: 
```
using Step = Int;
const Step ACTUAL_STEP = -1;
using ClassifierResult = map<Step, PDF>;
```
but it might be an overkill

Reply:
I will add:

`const Int ACTUAL_VALUE = -1;`

but not:

`using Step = Int;`